.pre-commit-config.yaml

```diff
@@ -51,23 +24,3 @@ repos:
     hooks:
     -   id: reorder-python-imports
         args: [--py37-plus]
```

Comment:
we shouldn't need this, our formatter can reorder imports already

Reply:
Thanks, added the option to get ruff to do it.